### PR TITLE
cp-209: Add hover effect to sidebar items

### PR DESCRIPTION
**Pull request by dmytro-kucherenko**
Done: 
- add hover effect to sidebar items

https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/ce810e16-58fe-4aa3-ac0e-bb8ba7c0338e

### CODE REVIEW

File: frontend/src/libs/components/sidebar/styles.module.scss [hunk 1]
```diff
@@ -45,16 +45,9 @@
...
-.icon-selected {
-  position: relative;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  background-color: var(--white);
-  border: 0;
-  border-radius: 50%;
+.icon-selected,
+.icon-container:hover {
```
Comment (OlesiaTr) [inline frontend/src/libs/components/sidebar/styles.module.scss]: We don't need the same effect on focus?

Comment (dmytro-kucherenko) [reply frontend/src/libs/components/sidebar/styles.module.scss]: We used identical hover and select effects on items of meditation and chat sidebars, I used the same approach. Personally I don't know how the hover effect should look like)

Comment (OlesiaTr) [reply frontend/src/libs/components/sidebar/styles.module.scss]:
```suggestion
.icon-container:hover, .icon-container:focus {
```

I meant just this, cause we supposed to have some effects when clicking with Tab for example

Comment (dmytro-kucherenko) [reply frontend/src/libs/components/sidebar/styles.module.scss]: Oh okay, I added it

**Review by liza-veis — DISMISSED** [review]

**Review by OlesiaTr — DISMISSED** [review]

**Review by fshabanov — DISMISSED** [review]

**Review by v0ldemar01 — DISMISSED** [review]

**Review by OlesiaTr — APPROVED** [review]

**Review by danilittaa — APPROVED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**